client/pages/sections/legacy-introduction-review.js

```diff
@@ -38,8 +39,17 @@ const LegacyIntroduction = ({ fields, project, values, pdf, readonly, title }) =
       )}
       <ReviewFields
         values={values}
-        fields={fields}
+        fields={fields.filter(f => f.name !== 'continuation')}
       />
+    {
+      values.continuation && (
+        <Review
```

Comment:
It seems like we've put a load of extra logic into the `<Review/>` component to handle the special case where the field is `continuation`, and it be much cleaner to leave the review component untouched and just put the bespoke content here.

Reply:
yes you're right, it _was_ just included in the `<ReviewFields />` component, but now we've called it out separately it would make sense to do that. I was totally out of fucks by that point